static/styles/left_sidebar.css

```diff
@@ -454,7 +455,6 @@ li.top_left_recent_topics {
     overflow-x: hidden;
     overflow-x: clip;
     text-overflow: ellipsis;
-    padding-right: 2px;
```

Comment:
Hmm, isn't the `conversation-partners` variant of this rule, above, for the PMs section and thus not in scope for the above?

An alternative idea would be to just remove the rule on the `a`  element above, and make this padding a variable `before-unread-count-padding` that gets used in these 3 places.

Reply:
Thanks for pointing that out! Have fixed this as suggested and also removed some repetitive CSS in the process (though I'm not sure if the variable was needed since it's used just once now 🤔)